src/style.css

```
@@ -1 +1,15 @@
-/* styles go here */
+.form-field {
+  display: block;
+}
+
+.radio {
+  display: inline-block;
+}
+
+.fieldset {
```

```suggestion
.fieldset {
```
Краще не давати класам ім'я, які співпадають з назвами тегів